backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -77,9 +80,21 @@ import { CacheService } from './CacheService';
 import { QueryService } from './QueryService';
 import { ArchivedStats } from '../models/ArchivedStats';
 import { ArchivedStatsRepository } from '../repositories/ArchivedStatsRepository';
+import { validate } from 'class-validator';
+import { plainToClass } from 'class-transformer';
+import { StratificationFactorRepository } from '../repositories/StratificationFactorRepository';
 
 @Service()
 export class ExperimentService {
+  backendVersion = env.app.version;
+  allIdMap = {};
+  errorRemovePart = 'instance of ExperimentDTO has failed the validation:\n - ';
```

Comment:
These hardcoded texts can be moved outside of the the class

Reply:
code updated